server/src/BitBakeDocScanner.ts

```diff
@@ -41,7 +41,7 @@ const variableInfosOverrides: Record<string, VariableInfosOverride> = {
   }
 }
 
-const variablesFolder = 'doc/bitbake-user-manual/bitbake-user-manual-ref-variables.rst'
+const variablesFilePath = 'doc/bitbake-user-manual/bitbake-user-manual-ref-variables.rst'
```

Comment:
Just to open up a discussion: You mention "the previous" completion items as fallback. Do we even want them?
If I'm not mistaken, Ross advised us to bundle against one version of poky doc inside our extension instead of scanning the user's doc. I think it's a great idea as well. We already have a npm fetch script that gets poky. We should statically generate the variables list from there. Can be the topic of another PR.

Reply:
Noted. Sounds like a good approach, we will then need a process to regularly update the docs in our code base (Probably in GitHub actions).